src/shared/utilities/childProcess.ts

```diff
@@ -108,4 +108,10 @@ export class ChildProcess {
             this.childProcess!.removeAllListeners()
         })
     }
+
+    public async kill(signal?: string): Promise<void> {
+        if (this.childProcess) {
+            this.childProcess.kill(signal)
```

Comment:
Can you check if the process has not closed first?

> Sending a signal to a child process that has already exited is not an error but may have unforeseen consequences. Specifically, if the process identifier (PID) has been reassigned to another process, the signal will be delivered to that process instead which can have unexpected results.

https://nodejs.org/docs/latest-v10.x/api/child_process.html#child_process_subprocess_kill_signal

Also, can a test be added, possibly verifying against childProcess.killed?

Reply:
✔️